content/en/docs/tutorials/security/apparmor.md

```diff
@@ -250,10 +250,11 @@ error: error executing remote command: command terminated with non-zero exit cod
 
 To wrap up, let's look at what happens if we try to specify a profile that hasn't been loaded:
 
-```shell
-kubectl create -f /dev/stdin <<EOF
-```
```

Comment:
Best way to do this is to switch to a `code` shortcode - see https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files and look at other pages to see how we make that work with `kubectl -f`.